test/letters/components/AddressBlock.unit.spec.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+import AddressBlock from '../../../src/js/letters/components/AddressBlock';
+
+const defaultProps = { name: 'Gary Todd' };
+
+describe('<AddressBlock/>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(<AddressBlock { ...defaultProps }/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.exist;
```

Comment:
Mmm...Can we check to make sure it's just got some text we expect it to have? That way we know for sure it actually rendered rather than there being DOM.

Maybe it's trivial, but I think we've always checked against some element we knew it should render to avoid it failing to render but passing the test.

Reply:
Yeah, I saw this convention being used elsewhere but it shouldn't be used as the sole test for a component. Fixing.